website/docs/guides/dynamic-loading-catalogs.md

```diff
@@ -4,60 +4,19 @@
 
 Here's an example of a basic setup with a dynamic load of catalogs.
 
-## Setup
-
-:::caution
-You don't have to install following Babel plugins if you're using *Create React App* or similar framework which already has it.
-:::
-
-We are using the [Dynamic Import() Proposal](https://github.com/tc39/proposal-dynamic-import) to ECMAScript. We need to install `@babel/plugin-syntax-dynamic-import` and `babel-plugin-dynamic-import-node` to make it work. Also, the code examples given here make use of `@babel/plugin-proposal-class-properties`.
-
-```bash npm2yarn
-npm install --save-dev \
-  @babel/plugin-syntax-dynamic-import \
-  babel-plugin-dynamic-import-node \
-  @babel/plugin-proposal-class-properties
-```
-
-:::caution
-`babel-plugin-dynamic-import-node` is required when running tests in Jest.
-:::
-
-``` js title=".babelrc"
-{
-  "plugins": [
-    "@babel/plugin-syntax-dynamic-import",
-    "@babel/plugin-proposal-class-properties"
-  ],
-  "env": {
-    "test": {
-      "plugins": [
-        "dynamic-import-node"
-      ]
-    }
-  }
-}
-```
-
```

Comment:
This is outdated, all mentioned plugins and syntaxes are already part of the language.